src/main/scala/it/pagopa/interop/catalogprocess/service/impl/CatalogManagementServiceImpl.scala

```diff
@@ -142,20 +173,12 @@ final case class CatalogManagementServiceImpl(invoker: CatalogManagementInvoker,
     invoker.invoke(request, s"Eservice $eServiceId descriptor $descriptorId has been moved to draft")
   }
 
-  private def updatableEservice(eService: EService): Future[EService] = Either
-    .cond(
-      eService.descriptors.length <= 1,
```

Comment:
this check has been moved [here](https://github.com/pagopa/interop-be-catalog-process/pull/166/files?diff=split&w=1#diff-3314a54a9a09d46d53883dde47ceedaaccda0ddbdfb0d25c85847b8de1951c74R72) and fixed.
As far as I can tell this version is wrong, it would allow to update an EService containing 1 valid descriptor (es Published) not in Draft